lib/l10n/intl_en.arb

```diff
@@ -906,6 +906,42 @@
   "demoSimpleDialogDescription": "A simple dialog offers the user a choice between several options. A simple dialog has an optional title that is displayed above the choices.",
   "@demoSimpleDialogDescription": {
     "description": "Description for the simple dialog component demo."
+  },
+    "demoDividerTitle": "Divider",
+  "@demoDividerTitle": {
+    "description": "Title for the divider component demo."
+  },
+  "demoDividerSubtitle": "A divider is a thin line that groups content in lists and layouts",
+  "@demoDividerSubtitle": {
+    "description": "Subtitle for the divider component demo."
+  },
+  "demoDividerDescription": "Dividers can be used in lists, drawers, and elsewhere to separate content.",
+  "@demoDividerDescription": {
+    "description": "Description for the divider component demo."
+  },
+  "demoDividerAbove": "Above",
+  "@demoDividerAbove": {
+    "description": "Text one for the divider component demo."
+  },
+  "demoDividerBelow": "Below",
+  "@demoDividerBelow": {
+    "description": "Text two for the divider component demo."
+  },
+  "demoVerticalDividerTitle": "Vertical Divider",
+  "@demoVerticalDividerTitle": {
+    "description": "Title for the vertical divider component demo."
+  },
+  "demoVerticalDividerDescription": "Dividers can be used in lists, drawers, and elsewhere to separate content.",
```

Comment:
Since this is the same as `demoDividerDescription`, just remove this one and re-use the other one